docs/src/main/sphinx/installation/cli.rst

```diff
@@ -116,6 +130,11 @@ JWT authentication
 To access a Trino cluster configured to use :doc:`/security/jwt`, use the
 ``--access-token=<token>`` option to pass a JWT to the server.
 
+Kerberos authentication
```

Comment:
Note for other reviewers .. we are doing this for now in this PR.. we soon get rid of the other docs and migrate all the info into here.. but one step at a time

Reply:
👣